autoPyTorch/datasets/base_dataset.py

```diff
@@ -13,36 +13,36 @@
 
 from autoPyTorch.constants import CLASSIFICATION_OUTPUTS, STRING_TO_OUTPUT_TYPES
 from autoPyTorch.datasets.resampling_strategy import (
-    CROSS_VAL_FN,
+    CrossValFunc,
+    CrossValFuncs,
```

Comment:
Do you think we can do something to improve the code so that we do not have 2 elements that only differ with an 's'?

What I mean is that, is there something we can further do to get rid of either `CrossValFunc` or `CrossValFuncs`?

Reply:
Yeah, I did it in the successive PR (#108)!